subsys/bluetooth/shell/bt.c

```diff
@@ -2401,11 +2411,13 @@ SHELL_STATIC_SUBCMD_SET_CREATE(bt_cmds,
 #if defined(CONFIG_BT_BROADCASTER)
 	SHELL_CMD_ARG(advertise, NULL,
 		      "<type: off, on, scan, nconn> [mode: discov, non_discov] "
-		      "[whitelist: wl, wl-scan, wl-conn] [identity] [no-name]",
-		      cmd_advertise, 2, 4),
+		      "[whitelist: wl, wl-scan, wl-conn] [identity] [no-name]"
```

Comment:
missing a space at the end of the string?

Reply:
Yup, fixed.